packages/front-end/components/ConfirmModal.tsx

```diff
@@ -51,7 +51,7 @@ const ConfirmModal = ({
         aria-labelledby="exampleModalLabel"
         aria-hidden="true"
       >
-        <div className="modal-dialog" role="document">
+        <div className="modal-dialog modal-dialog-centered" role="document">
```

Comment:
I find the confirmation modal looks better when it is vertically centered on the page.  I checked out the two other places where we have it (invite delete and presentation delete) and at least to me it looks better vertically centered there too.